docs/index.md

```diff
@@ -201,6 +201,46 @@ resource "grafana_oncall_escalation" "example_notify_step" {
 }
 ```
 
+Alternatively, you can also configure the provider block by setting `url`
+to your Grafana URL and `auth` to a service account token:
+
+```terraform
+// Step 1: Configure provider block.
+provider "grafana" {
+  alias = "oncall"
+  url   = "http://grafana.example.com/"
```

Comment:
interesting, so we no longer need to make the Terraform provider "aware" of the OnCall API url?

Reply:
Not really, but there was always a default value for it (`prod-us-central-0`), so you can omit setting it in some cases (see existing [docs](https://registry.terraform.io/providers/grafana/grafana/latest/docs#managing-grafana-oncall)). It would be great not to need this URL, but there isn't an easy way to figure it out atm (afaict).

Reply:
> It would be great not to need this URL, but there isn't an easy way to figure it out atm

In hosted grafana land we could _maybe_ call the GCOM region API to fetch your stack's oncall regional API url? but yeah this won't support OSS